.github/workflows/esti.yaml

```diff
@@ -974,9 +974,15 @@ jobs:
           ESTI_AZURE_STORAGE_ACCOUNT: esti4hns
           ESTI_AZURE_STORAGE_ACCESS_KEY: ${{ secrets.LAKEFS_BLOCKSTORE_AZURE_STORAGE_GEN2_ACCESS_KEY }}
 
+      - name: Az CLI login
+        if: always()
+        uses: azure/login@v1
+        with:
+          client-id: ${{ secrets.AZURE_COSMOSDB_CLEANUP_CLIENT_ID }}
```

Comment:
Just so I understand - why can't we use our regular AZURE_CLIENT_ID here?

Reply:
That's the first thing I tried. But that principal/identity doesn't have GH federated OIDC enabled, and I don't know which one it is because I can't view the GH secret. I ended up re-using a managed identity that's already used for other esti-related things.